src/components/Router.js

```diff
@@ -75,7 +76,7 @@ export default class Router extends Component {
       ({ pathname, search }) => this.setState({ pathname, search })
     )
 
-    if (isFirecloud() && new URLSearchParams(window.location.search).has('fcredir')) {
+    if (isFirecloud() && _.has('fcredir', qs.parse(Nav.history.location.search, { ignoreQueryPrefix: true, plainObjects: true }))) {
```

Comment:
is this backwards? should it be `_.has(qs.parse(...), 'fcredir')`?

Reply:
We use the `fp` version of lodash, which switches the arguments.

Reply:
ah, gotcha, thanks!